specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/backuprestore.json

```
@@ -82,6 +82,63 @@
         }
       }
     },
+    "/prebackup": {
+      "post": {
+        "tags": [
+          "PreFullBackup"
+        ],
+        "operationId": "PreFullBackup",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "description": "Pre backup operation for checking whether the customer can perform a full backup operation or not.",
```

```suggestion
        "description": "Pre-backup operation for checking whether the customer can perform a full backup operation.",
```

"or not" is redundant with "checking whether".